pkg/tenant/members.go

```diff
@@ -184,11 +184,23 @@ func (s *Server) MemberList(c *gin.Context) {
 func (s *Server) MemberCreate(c *gin.Context) {
 	var (
 		err    error
+		m      *tokens.Claims
 		member *api.Member
-		out    *api.Member
 	)
 
-	// TODO: Add authentication middleware to fetch the organization ID.
+	// Fetch member from the context.
+	if m, err = middleware.GetClaims(c); err != nil {
+		log.Error().Err(err).Msg("could not fetch member from context")
+		c.JSON(http.StatusUnauthorized, api.ErrorResponse(err))
+		return
+	}
+
+	// Get the member's organization ID and return a 500 response if it is not a ULID.
+	if _, err = ulid.Parse(m.OrgID); err != nil {
```

Comment:
The Member model takes in a Tenant ID, not an OrgID and that is why the value isn't captured here and used elsewhere.

Reply:
I see, since this isn't going to work for right now (the members and projects models need to be fleshed out as you pointed out) can we add a check at the top of this method and in `ProjectCreate` so that this returns not implemented?

```golang
	if pkg.VersionReleaseLevel == "alpha" {
		c.JSON(http.StatusNotImplemented, api.ErrorResponse("not implemented"))
		return
	}
```

Then once we're past the alpha release we can go back and fix these. Also we should probably add a `suite.T().Skip()` to the corresponding tests so they are skipped for now.